README.md

```diff
@@ -32,17 +32,21 @@ tuxedo-control-center
 
    sudo apt install -y git gcc g++ make nodejs
    ```
-2. Clone & install libraries
+
+2. Install `tuxedo-cc-wmi` (can be obtained from the respositories [here](https://www.tuxedocomputers.com/en/Infos/Help-and-Support/Instructions/Add-TUXEDO-Computers-software-package-sources.tuxedo#)).
+   This package provides a driver at `/dev/tuxedo_cc_wmi`. The control center can run without this but will not be functional.
```

Comment:
Is there a dependency list to add it to?

Reply:
I'm not sure I understand the question. I just thought I would mention that if you are building the control center from source then you need to install the driver separately. Previously there was no mention of that in the README.